pipelines/azureml/conf/experiments/lightgbm_training_sweep/cpu.yaml

```diff
@@ -0,0 +1,67 @@
+# to execute, run from /pipelines/azureml/
+# > python pipelines/lightgbm_training.py --config-dir ./conf --config-name experiments/lightgbm_training_sweep/cpu run.submit=True
+
+defaults:
+  - aml: sample
+  - compute: sample
+  - modules: benchmark
+
+# run parameters are command line arguments for running your experiment
+run: # params for running pipeline
+  experiment_name: "lightgbmbenchmark-training" # IMPORTANT
+  regenerate_outputs: false
+  continue_on_failure: false
+  verbose: false
+  submit: false
+  resume: false
+  canary: false
+  silent: false
+  wait: false
+
+module_loader: # module loading params
+  use_local: "*"
+  force_default_module_version: null
+  force_all_module_version: null
+  local_steps_folder: "../../../../../src/scripts/"
+
+### CUSTOM PARAMETERS ###
+
+lightgbm_training:
+  benchmark_name: "benchmark-dev"
+
+  # DATA
+  learning_task: "regression"
+  train_samples: 100000
+  test_samples: 10000
+  inferencing_samples: 10000
+  n_features: 4000
+  n_informative: 4000
+
+  # TRAINING
+  # fixed training parameters
+  objective: "regression"
+  metric: "rmse"
+  boosting: "gbdt"
+  tree_learner: "data"
+
+  # "sweepable" training parameters
```

Comment:
@ruizhuanguw you might want to provide a couple example values here.

Reply:
Created an issue #77 to build on this PR, I think this PR is ok to merge when reviewed, and we can iterate in another PR on the perfect settings.